pkgs/servers/windmill/default.nix

```diff
@@ -0,0 +1,89 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, callPackage
+, pkg-config
+, openssl
+, python3
+, deno
+, librusty_v8 ? callPackage ../../development/web/deno/librusty_v8.nix { }
+, swagger-cli
+, nsjail
+, lld
+, makeWrapper
+, rustfmt
+, go
+, bash
+}:
+
+let
+  pname = "windmill";
+  version = "1.56.1";
+  fullSrc = fetchFromGitHub {
```

Comment:
```suggestion
  pname = "windmill";
  version = "1.56.1";
  src = fetchFromGitHub {
```
also please inline

Reply:
Cannot inline this as we need one file from `fullSrc` later. `sourceRoot` does not work as we need to move the frontend files to the top level. Let me know if there is a more elegant solution.